360-Videos-Metadata/360VideosMetadata.py

```diff
@@ -58,18 +58,27 @@
 # XML contents.
 rdf_prefix = " xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\" "
 
-spherical_xml = (
+spherical_xml_header = (
     "<?xml version=\"1.0\"?>\n"
```

Comment:
up to you if you like it better, but you can use multiline strings instead

spherical_xml_header = \
"""<?xml ....
blah
blah"""